packages/react/test/specs/components/Attachment/Attachment-test.tsx

```diff
@@ -18,5 +21,32 @@ describe('Attachment', () => {
     handlesAccessibility(Attachment, {
       defaultRootRole: undefined,
     })
+
+    test('handleClick is executed when Enter is pressed for attachment, not for more option button', () => {
+      const onClickAttachment = jest.fn()
+      const onClickButton = jest.fn()
+      const attachement = mountWithProvider(
+        <Attachment
+          actionable
+          action={{
+            icon: 'more',
+            onClick: () => onClickButton(),
+          }}
+          onClick={onClickAttachment}
+        />,
+      )
+      const moreOptionButton = attachement.find('button')
```

Comment:
try to use a class selector rather than type of element.

Reply:
probably you need to add slot class names to `Attachment`, and use the one you will add for `action` here as class selector.

Reply:
I cannot see difference if button is found by 'type of element' or by the class. What is advantage of that? and why I should adding slot class name into Attachment only because the test?